content/c-sharp/concepts/enums/enums.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'Enums'
+Description: 'An enum (or "Enumeration type") is a set of named labels that each represent a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Enums'
+  - 'Data Types'
+  - 'Arrays'
+  - 'Vectors'
+  - 'Pointers'
+  - 'Memory'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
```

Comment:
We need an intro paragraph that summarizes what an enum is. It should at the very least be a copy of the description above. (With the first occurrence of the term "enum" in bold.)